app/src/main/java/de/danoeh/antennapod/fragment/StatisticsFragment.java

```diff
@@ -84,8 +90,7 @@ private void loadStats() {
 
     private String timeToText(long time) {
         int hours = (int) time / 3600;
-        int remainder = (int) time - hours * 3600;
-        int mins = remainder / 60;
+        int mins = (int) time % 3600;
```

Comment:
Oh sorry... I added my own method to `Converter` because it contains the word "hours". Appending it manually every time is so much additional code